apps/storefront/src/components/RegisteredCloseButton.tsx

```diff
@@ -20,6 +20,8 @@ export default function RegisteredCloseButton(props: CloseButtonProps) {
   const navigate = useNavigate();
 
   const handleCloseForm = () => {
+    const isInLoginPage = window.location.href.includes('#/login');
```

Comment:
🤔 do we want to check for the hash?

`window.location.hash.startsWith('#/login')`

Reply:
Here is to determine whether it is on the login page

Reply:
yes, all I am saying is you can use the `location.hash` instead of `location.href`, and it makes the check easier

Reply:
Well, I have made the replacement, thanks for your suggestion.